### PR TITLE
Backend setup servlet files created

### PR DESCRIPTION
**Pull request by Ionis-Kutrolli**
Creation of servlet files the frontend will use to get information.

### CODE REVIEW

File: src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.google.impactdashboard.api_utilities;
+
+import com.google.cloud.logging.v2.LoggingClient;
+
```
Comment (peterpie123) [inline src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java:4]: super nit: no newlines between import statements https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

File: src/main/webapp/WEB-INF/cron.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cron:
+- description: "daily update to the database"
+  url: /update-data
+  schedule: every 24 hours
```
Comment (peterpie123) [inline src/main/webapp/WEB-INF/cron.yaml:4]: nit: newline at EOF ;)

File: src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java (hunk 1 above)
Comment (Ionis-Kutrolli) [reply src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java:4]: Ops I think that was a setting predefined in intellij I'll fix that

File: src/main/java/com/google/impactdashboard/api_utilities/RecommendationRetriever.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package com.google.impactdashboard.api_utilities;
+
+import com.google.cloud.recommender.v1beta1.ListRecommendationsRequest;
```
Comment (yutaow) [inline src/main/java/com/google/impactdashboard/api_utilities/RecommendationRetriever.java:3]: I thought recommender API is already in GA. Any reason of using v1beta1?

Comment (Ionis-Kutrolli) [reply src/main/java/com/google/impactdashboard/api_utilities/RecommendationRetriever.java:3]: There is not reason currently. The only reason I chose v1beta1 over v1 is the possibility of using the insight classes provided in beta1 in the future.

Comment (Ionis-Kutrolli) [reply src/main/java/com/google/impactdashboard/api_utilities/RecommendationRetriever.java:3]: I'll change this to v1 until we know if we need v1beta1

File: src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import java.io.IOException;
+import java.util.Collection;
+
+public class LogRetriever {
+
+  /** Cloud Logging client used to retrieve Audit logs and Recommendation logs */
+  private LoggingClient logger;
+
+  public LogRetriever() throws IOException {
```
Comment (yutaow) [inline src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java:14]: Prefer factory here too.

File: src/main/webapp/WEB-INF/cron.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+cron:
+- description: "daily update to the database"
```
[the remark this answers is not in the document]
Comment (Ionis-Kutrolli) [reply src/main/webapp/WEB-INF/cron.yaml:2]: After looking into it a bit further it seems like there doesn't need to be a dash there.

**Pull request merged**